app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils
+
+DESCRIPTION="A tool to handle your cellular phone"
```

Comment:
Please keep the variables in devmanual order, i.e.

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

DEPEND/RDEPEND/...
```

Most of people working on ebuilds are used to this order, and it makes it easier to locate stuff.

Reply:
sorry, i sort they too :)